tests/unit/plugins/test_enricher.py

```diff
@@ -19,18 +19,132 @@
 import pytest  # NOQA
 from gordon import interfaces
 
+from gordon_gcp import exceptions
 from gordon_gcp.plugins import enricher
+from gordon_gcp.plugins import event_consumer
 
 
-def test_implements_interface():
+INSTANCE_NETWORK_DATA = {
+    'networkInterfaces': [{
+        'accessConfigs': [{
+            'natIP': '35.229.173.136'
+        }],
+    }]
+}
+
+
+@pytest.fixture
+def config(fake_keyfile):
+    return {
+        'keyfile': fake_keyfile,
+        'scopes': [],
+        'dns_zone': 'example.com',
+        'retries': 5
+    }
+
+
+def test_implements_interface(config):
     """GCEEnricher implements IEnricherClient"""
-    config = {'foo': 'bar'}
     success, error = asyncio.Queue(), asyncio.Queue()
-    client = enricher.GCEEnricher(config, success, error)
+    client = enricher.GCEEnricher(config, None, success, error)
 
     assert interfaces.IEnricherClient.providedBy(client)
     assert interfaces.IEnricherClient.implementedBy(enricher.GCEEnricher)
     assert config is client.config
     assert success is client.success_channel
     assert error is client.error_channel
     assert 'enrich' == client.phase
+
+
+@pytest.fixture
+def event_msg(mocker):
```

Comment:
Maybe this could be moved to `conftest.py` since I also have a fixture like this in the event consumer plugin.